ONSdigital/dockers
# Move to checking sha256 on pip modules

**Changes**
- Check sha on all dependencies and downstream dependencies
- Add the sha for modules to requirements.txt

**How to test**
- Build the dockerfile in the flask-crypto directory and make sure it doesn't fail
